### PR TITLE
docs on gitea usage for catalog

### PR DESCRIPTION
**Pull request by takirala**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-82742

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/index.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ DKP applications are catalog applications provided by D2iQ for use in your envir
 
 Follow these steps to install the DKP catalog from the CLI.
 
+<<<<<<< HEAD
```
Comment (vacas5) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/index.md:15]: looks like a conflict got leftover somehow

**Review by vacas5 — CHANGES_REQUESTED** [review]

File: pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
+---
+layout: layout.pug
+navigationTitle: Host on Gitea
+title: Host on Gitea
+menuWeight: 10
+excerpt: How to host git repositories on Gitea for air gapped deployments
+---
+
+DKP supports using an external catalog git repository to install applications. In cases where an external repository is not accessible from within the cluster, you can use the Gitea server that comes out-of-box with a Kommander deployment to install applications.
```
Comment (vacas5) [inline pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md:9]: can you add Gitea to the vocab?

File: pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+## Prerequisites
+
+1. `git` needs to be installed in the local environment (from where a connection to air-gapped cluster can be established)
```
Comment (vacas5) [inline pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md:13]: how about "install `git` in the local environment"?

**Review by vacas5 — CHANGES_REQUESTED** [review]
looks like there are a lot of good vale recommendations for fixing the grammar

File: pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
+---
+layout: layout.pug
+navigationTitle: Host on Gitea
+title: Host on Gitea
+menuWeight: 10
+excerpt: How to host git repositories on Gitea for air gapped deployments
+---
+
+DKP supports using an external catalog git repository to install applications. In cases where an external repository is not accessible from within the cluster, you can use the Gitea server that comes out-of-box with a Kommander deployment to install applications.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md:9]:
```suggestion
DKP supports using an external catalog git repository to install applications. In cases where an external repository is not accessible from within the cluster, to install applications, use the Gitea server that comes out-of-box with a Kommander deployment.
```

File: pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+## Prerequisites
+
+1. Install `git` in the local environment (from where a connection to air-gapped cluster is established).
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md:13]:
```suggestion
- Install `git` in the local environment (from where a connection to air-gapped cluster is established).

```

File: pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+## Host external repository on Gitea
+
+1. Set the `VERSION` environment variable to the version of Kommander you want to install, for example:
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md:17]:
```suggestion
To use the Gitea server that comes with a Kommander deployment, follow these steps:

1. Set the `VERSION` environment variable to the version of Kommander you want to install, for example:
```

File: pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+      ref:
+        branch: ${GITEA_REPOSITORY_DEFAULT_BRANCH}
+      timeout: 20s
+      url: https://${GITEA_HOSTNAME}/dkp/kommander/git/${GITEA_USERNAME}/${GITEA_REPOSITORY_NAME}
+      secretRef:
+        name: ${TARGET_NAMESPACE}
+    EOF
+    ```
+
+1. Once the newly created `GitRepository` on the management cluster is reconciled, corresponding `App`s are loaded by Kommander controller.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/install/air-gapped/catalog/index.md:108]:
```suggestion
1. After the newly created `GitRepository` on the management cluster reconciles, any corresponding `App`s are loaded by Kommander controller.
```

File: pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-applications/add-create-git-repo/index.md [hunk 8]
```diff
@@ -8,6 +8,8 @@ excerpt: Create a Git Repository in the Project namespace
 
 Use the CLI to create the GitRepository resource and add a new repository to your Project.
 
+1.  If running in air-gapped environment, refer to [air-gapped setup instructions](../../../../../install/air-gapped/catalog).
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-applications/add-create-git-repo/index.md:11]:
```suggestion
1.  Refer to [air-gapped setup instructions](../../../../../install/air-gapped/catalog), if you are running in air-gapped environment.
```

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/index.md [hunk 9]
```diff
@@ -12,6 +12,8 @@ DKP applications are catalog applications provided by D2iQ for use in your envir
 
 Follow these steps to install the DKP catalog from the CLI.
 
+1. If running in air-gapped environment, refer to [air-gapped setup instructions](../../../../install/air-gapped/catalog).
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/index.md:15]:
```suggestion
1. Refer to [air-gapped setup instructions](../../../../install/air-gapped/catalog), if you are running in air-gapped environment.
```

**Review by Colin1841 — APPROVED** [review]

**Comment by takirala** [comment]
@vacas5 can you PTAL

**Review by vacas5 — APPROVED** [review]
you didn't just make Vale happy, you made ME happy :)

**Pull request merged**